client/test/unit/specs/transactions.spec.js

```diff
@@ -312,11 +312,11 @@ describe('Testing Transactions.vue EXPIRED', () => {
     // set NR info
     vm.$store.commit('setNrInfo', {
       additionalInfo: 'The quick brown fox...',
-      consent_dt: '2021-08-01T07:01:00+00:00',
+      consent_dt: '2021-08-01T06:59:00+00:00',
       consentFlag: 'R',
       corpNum: 'BC1234567',
       entity_type_cd: null,
-      expirationDate: '2021-09-01T07:01:00+00:00',
+      expirationDate: '2021-09-01T06:59:00+00:00',
```

Comment:
Are there any tests in here for PST offset (8h) ?

And maybe there should be tests for the edge cases (ie, [the dates the time zone changes](https://en.wikipedia.org/wiki/Pacific_Time_Zone#:~:text=Effective%20in%20the%20U.S.%20in,the%20first%20Sunday%20in%20November.)).

Reply:
no there werent any tests before for PST offset, or time zone changes

Reply:
time zone changes will be difficult to test, because they appear on different days every year, so i would leave that one... but i can update one of the tests to include 07:59 instead of 06:59 if you would like

Reply:
You don't need a generic test for all possible change dates -- you can add tests for date changes in late 2021 and early 2022 :)